### PR TITLE
pc-i440fx: remove all references to rhel6

### PR DESCRIPTION
**Pull request by dominikholler**
All references to rhel6 are removed from
rhel6-server-large-pc-i440fx.yaml, because this template is
intended to be used for guests which are not compatible with
the q35 machine type. For rhel6 guests, the rhel6 templates
from common-templates should be used.

@ksimon1 please have a look.

### CODE REVIEW

**Comment by dominikholler** [comment]
I verified that the proposed code change works on OpenShift 4.9 .

File: templates/pc-i440fx/README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+Virtualization 2.6.X kubevirt-config Config Map should be updated.
+Open the kubevirt-config config map for editing by running the following
+command:
+
+`$ oc edit configmap kubevirt-config -n openshift-cnv`
+
+Edit the config map:
+
+```
+kind: ConfigMap
```
Comment (ksimon1) [inline templates/pc-i440fx/README.md:31]: This way of updating kubevirt config is deprecated and we should configure kubevirt via CR

Comment (dominikholler) [reply templates/pc-i440fx/README.md:31]: This section should be only about OpenShift < 4.8. Is the CR way already available in 2.6?

**Comment by dominikholler** [comment]
@vladikr Can you please have a look?

File: templates/pc-i440fx/README.md (hunk 1 above)
Comment (vladikr) [reply templates/pc-i440fx/README.md:31]: I think HCO is not updating the KubeVirt CR in 2.6

**Comment by vladikr** [comment]
Looks good to me. Thanks @dominikholler 
/lgtm

**Comment by dominikholler** [comment]
@vladikr and @ksimon1 thanks a lot for having a look! I removed the UI related content, so the template will only work on command line.
@phoracek Can you please merge?

**Pull request merged**